#################################
# PID parameters for pose seeking
#################################

# Input low-pass filter
input_low_pass_filter_half_order: 3
input_low_pass_filter_cutoff_frequency: 20.0

# Maximum value of error integral for all PID controllers
windup_limit: 0.05

# PID gains
x_proportional_gain: 30.0
y_proportional_gain: 30.0
z_proportional_gain: 30.0
x_integral_gain: 1.0
y_integral_gain: 1.0
z_integral_gain: 1.0
x_derivative_gain: 0.0
y_derivative_gain: 0.0
z_derivative_gain: 0.0

angular_proportional_gain: 10.0
angular_integral_gain: 1.0
angular_derivative_gain: 0.0
